gh-pages/index.html

```diff
@@ -368,7 +368,6 @@ <h2>Community Library Ports</h2>
   </div>
   <!-- content -->
 
-  <script src="assets/js/version.js" type="text/javascript"></script>
```

Comment:
This is already derived from `wanakana.VERSION` and this asset no longer exists.